src/app/ApplicationTemplate.Droid/ApplicationTemplate.Droid.csproj

```diff
@@ -1,254 +1,254 @@
+		<BundleAssemblies>false</BundleAssemblies>
+		<AndroidPackageFormat>aab</AndroidPackageFormat>
+		<AndroidUseAapt2>true</AndroidUseAapt2>
+	</PropertyGroup>
+	<Choose>
+		<When Condition="'$(IsLightBuild)' == 'True'">
+			<PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
+				<!-- Build optimization for lightbuild (faster builds in pull requests)-->
+				<AndroidEnableProfiledAot>false</AndroidEnableProfiledAot>
+				<AotAssemblies>false</AotAssemblies>
+				<EnableLLVM>false</EnableLLVM>
+				<Optimize>false</Optimize>
+			</PropertyGroup>
+		</When>
+		<Otherwise>
+			<PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
+				<AndroidEnableProfiledAot>true</AndroidEnableProfiledAot>
+				<AotAssemblies>true</AotAssemblies>
+				<EnableLLVM>true</EnableLLVM>
+				<Optimize>true</Optimize>
+			</PropertyGroup>
+		</Otherwise>
+	</Choose>
+	<ItemGroup>
+		<Reference Include="Mono.Android" />
+		<Reference Include="Mono.Android.Export" />
+		<Reference Include="mscorlib" />
+		<Reference Include="System" />
+		<Reference Include="System.Core" />
+		<Reference Include="System.Net.Http" />
+		<Reference Include="System.Xml.Linq" />
+		<Reference Include="System.Xml" />
+	</ItemGroup>
+	<ItemGroup>
+		<PackageReference Include="MessageDialogService.Uno" Version="0.5.0-feature.update-dotnet6.29" />
+		<PackageReference Include="Uno.Toolkit.UI" Version="1.4.0-dev.16" />
+		<PackageReference Include="Xamarin.AndroidX.Lifecycle.LiveData" Version="2.1.0" />
+		<PackageReference Include="Xamarin.Build.Download" Version="0.10.0" />
+		<PackageReference Include="Serilog.Sinks.Xamarin" Version="0.1.37" />
+		<PackageReference Include="Serilog.Sinks.File" Version="5.0.0" />
+		<PackageReference Include="Reactive.Annex.Uno" Version="0.5.0-feature.dotnet6.38" />
+		<PackageReference Include="Uno.Material" Version="1.3.3" />
+		<PackageReference Include="Uno.Microsoft.Toolkit.Uwp.DeveloperTools" Version="7.1.11" />
+		<PackageReference Include="Uno.Microsoft.Xaml.Behaviors.Uwp.Managed" Version="2.3.0" />
+		<PackageReference Include="Uno.UI" Version="4.4.20" />
+		<PackageReference Include="Uno.UI.RemoteControl" Version="4.4.20" />
+		<PackageReference Include="Uno.UniversalImageLoader" Version="1.9.36" />
+		<PackageReference Include="Chinook.DynamicMvvm.Uno" Version="1.1.0" />
+		<PackageReference Include="Chinook.SectionsNavigation.Uno" Version="1.1.0" />
+		<PackageReference Include="Chinook.DataLoader.Uno" Version="1.1.0" />
+		<PackageReference Include="Chinook.BackButtonManager.Uno" Version="1.1.1" />
+		<PackageReference Include="Nventive.AsyncWebView.Uno" Version="0.4.0-dev.22" />
+		<PackageReference Include="Nventive.ExtendedSplashScreen.Uno" Version="0.3.0-dev.38" />
+		<PackageReference Include="Nventive.Persistence.Uno" Version="0.2.0-feature.uno-ui-4.21" />
+		<PackageReference Include="Nventive.View.Uno" Version="0.4.0-feature.dotnet6.50" />
```

Comment:
There are still updates being made for that package, no? This update should wait for the real release. Same comments apply to the packages that use feature-branch versions. Just finish the work on the packages and use the ones generated from the main branch.

Reply:
I thought that this PR was to make sure the changes we did on the updated packages won't break the current template for Uno. We could change the target branch to a feature one to make sure main won't be affected

Reply:
The goal of the PR is good. Just make it a Draft PR until all packages are finished :)